web/templates/pages/list_db.html

```diff
@@ -83,102 +150,183 @@
+  $value) { ++$i; if ($data[$key]['SUSPENDED'] == 'yes') { $status = 'suspended'; $spnd_action =
+  'unsuspend'; $spnd_icon = 'fa-play'; $spnd_confirmation = _('UNSUSPEND_DATABASE_CONFIRMATION') ; }
+  else { $status = 'active'; $spnd_action = 'suspend'; $spnd_icon = 'fa-pause'; $spnd_confirmation =
+  _('SUSPEND_DATABASE_CONFIRMATION') ; } if ($data[$key]['HOST'] != 'localhost' ) $http_host =
+  $data[$key]['HOST']; if ($data[$key]['TYPE'] == 'mysql') $db_admin = "phpMyAdmin"; if
+  ($data[$key]['TYPE'] == 'mysql') $db_admin_link = "https://".$http_host."/phpmyadmin/"; if
+  (($data[$key]['TYPE'] == 'mysql') && (!empty($_SESSION['DB_PMA_ALIAS']))) $db_admin_link =
+  $_SESSION['DB_PMA_ALIAS']; if ($data[$key]['TYPE'] == 'pgsql') $db_admin = "phpPgAdmin"; if
+  ($data[$key]['TYPE'] == 'pgsql') $db_admin_link = "https://".$http_host."/phppgadmin/"; if
+  (($data[$key]['TYPE'] == 'pgsql') && (!empty($_SESSION['DB_PGA_ALIAS']))) $db_admin_link =
+  $_SESSION['DB_PGA_ALIAS']; ?>
+  <div
+    class="l-unit <?php if($status == 'suspended') echo 'l-unit--suspended'; ?> animate__animated animate__fadeIn"
+    v_unit_id="<?=$key?>"
+    v_section="db"
+    sort-date="<?=strtotime($data[$key]['DATE'].' '.$data[$key]['TIME'])?>"
+    sort-name="<?=$key?>"
+    sort-disk="<?=$data[$key]['U_DISK']?>"
+    sort-user="<?=$data[$key]['DBUSER']?>"
+    sort-server="<?=$data[$key]['HOST']?>"
+    sort-charset="<?=$data[$key]['CHARSET']?>"
+  >
+    <div class="l-unit__col l-unit__col--right">
+      <div>
+        <div class="clearfix l-unit__stat-col--left super-compact">
+          <input
+            id="check<?=$i ?>"
+            class="ch-toggle"
+            type="checkbox"
+            title="<?=_('Select');?>"
+            name="database[]"
+            value="<?=$key?>"
+            <?="$display_mode;?"
+          />>
+        </div>
+        <div class="clearfix l-unit__stat-col--left wide-3 truncate">
+          <?php if (($read_only === 'true') || ($data[$key]['SUSPENDED'] == 'yes')) {?>
+          <b><?=$key?></b>
+          <?php } else { ?>
+          <b
+            ><a
+              href="/edit/db/?database=<?=$key?>&token=<?=$_SESSION['token']?>"
+              title="<?=_('Editing Database');?>: <?=$key?>"
+              ><?=$key?></a
+            ></b
+          >
+          <?php } ?>
+        </div>
+        <!-- START QUICK ACTION TOOLBAR AREA -->
+        <div class="clearfix l-unit__stat-col--left text-right compact-3">
+          <div class="l-unit-toolbar__col l-unit-toolbar__col--right u-noselect">
+            <div class="actions-panel clearfix">
+              <?php if ($read_only === 'true') {?>
+              <!-- Restrict the ability to edit, delete, or suspend domain items when impersonating 'admin' user -->
+              &nbsp;
+              <?php } else { ?>
+              <?php if ($data[$key]['SUSPENDED'] == 'no') {?>
+              <div class="actions-panel__col actions-panel__logs shortcut-enter" key-action="href">
```

Comment:
Looks like we're losing indentation here.

Reply:
Should be fixed now with file renaming